contracts/babylon/README.md

```diff
@@ -0,0 +1,62 @@
+# Babylon Smart Contract
+
+The `babylon` contract is the main entry point for Cosmos BSN Staking
+integration.
+It is responsible for handling the IBC messages and relaying them to the
+appropriate contract on the BSN side.
+
+## Interfaces
+
+This section provides a detailed overview of the interfaces provided by the
+contract.
+It only describes the interfaces that are relevant for Cosmos BSN Staking
+integration, and does not cover the full functionality of the contract.
+It also does not cover the different queries.
+
+### IBC Messages
```

Comment:
This is good for now, buy maybe at a later iteration of this document we could add more concrete information, such as the exact protobuf deifnitions of these messages. Wdyt?